packages/firestore/test/unit/specs/spec_builder.ts

```diff
@@ -1095,9 +1097,13 @@ export class SpecBuilder {
   private addQueryToActiveTargets(
     targetId: number,
     query: Query,
-    resumeToken?: string,
-    readTime?: TestSnapshotVersion
+    resume?: { resumeToken?: string; readTime?: TestSnapshotVersion },
+    expectedCount?: number
```

Comment:
Consider adding `expectedCount` into the `resume` object, instead of having it as a separate argument.